packages/frontend-2/pages/settings/workspaces/[slug]/security.vue

```diff
@@ -116,9 +110,9 @@
     </div>
 
     <SettingsWorkspacesSecurityDomainRemoveDialog
-      v-if="removeDialogDomain"
+      v-if="removeDialogDomain && workspace"
```

Comment:
ditto, every time we do v-if="workspace" on a component we're making the components load in serially, so if they have their own queries they're executed serially instead of in parallel